packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -755,15 +755,36 @@ const setOutputValues = async () => {
 const preProcessedInterventionsData = computed<Dictionary<{ name: string; value: number; time: number }[]>>(() => {
 	const state = _.cloneDeep(props.node.state);
 
-	const data = state.interventionPolicyGroups.flatMap((ele) =>
-		ele.intervention.staticInterventions.map((intervention) => ({
-			name: ele.intervention.appliedTo,
-			value: intervention.value,
-			time: intervention.timestep
+	// Combine before and after interventions
```

Comment:
Are you using this to combine the input interventions along side the output of optimize?
Please see `createInterventionPolicyFromOptimize` this may be useful here

Reply:
I'm using the response from that `createInterventionPolicyFromOptimize`  function to store the new optimized interventions object in the state (will be changed based on dans comment)!  Im just mapping the data for the before and after optimize interventions to create the chart markers here.

Reply:
Does it make sense to pass both of pre and post to the intervention chart's component and let it handle this rather than having this in the drilldown? 
You will know better for this so either way up to you! 
If you do want to do this a follow up PR similar to the state update makes total sense!

Reply:
I might take a look in a followup!